Formula/terraform-provisioner-ansible.rb

```diff
@@ -0,0 +1,42 @@
+require "language/go"
+
+class TerraformProvisionerAnsible < Formula
+  desc "Ansible provisioner for Terraform"
+  homepage "https://github.com/jonmorehouse/terraform-provisioner-ansible"
+  url "https://github.com/jonmorehouse/terraform-provisioner-ansible/archive/0.0.2.tar.gz"
+  sha256 "5ba93ea5d6f28e156d7e4eba90cd89032f7dc6aff396a8429c903da4efa2d8b6"
+  head "https://github.com/jonmorehouse/terraform-provisioner-ansible.git"
+
+  depends_on "go" => :build
+
+  terraform_deps = %w[
+    github.com/hashicorp/terraform fa6a83ebdc323f2b415779786e102e69ddbf9a48
+    github.com/mitchellh/mapstructure f3009df150dadf309fdee4a54ed65c124afad715
+    github.com/mitchellh/go-homedir 756f7b183b7ab78acdbbee5c7f392838ed459dda
+    github.com/mitchellh/go-linereader 07bab5fdd9580500aea6ada0e09df4aa28e68abd
+  ]
+
+  terraform_deps.each_slice(2) do |x, y|
+    go_resource x do
+      url "https://#{x}.git", :revision => y
+    end
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    terrapath = buildpath/"src/github.com/jonmorehouse/terraform-provisioner-ansible"
+    terrapath.install Dir["*"]
+
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    cd terrapath do
+      system "go", "build"
+      bin.install "terraform-provisioner-ansible"
+    end
+  end
+
+  test do
```

Comment:
I have not been able to do that properly since it requires a file in the home directory to work properly and I don'T know another way than just calling `terraform` without any path to get it executed.